Packs/Slack/Integrations/SlackV3/SlackV3_test.py

```diff
@@ -3291,26 +3289,53 @@ def test_send_message_to_destinations(mocker):
     assert args['body']['text']
 
 
-def test_send_file_to_destinations(mocker):
+def test_send_file_to_destinations_request_args(mocker):
+    """
+    Given:
+        - Sending file to a channel
+    When:
+        - Calling the send_slack_request_sync function
+    Then:
+        - Assert the function is called once and the correct arguments are sent to the function
+    """
     import SlackV3
     # Set
-
-    link = 'https://www.eizelulz.com:8443/#/WarRoom/727'
-    mocker.patch.object(demisto, 'investigation', return_value={'type': 1})
-    mocker.patch.object(demisto, 'demistoUrls', return_value={'warRoom': link})
-    mocker.patch.object(demisto, 'getIntegrationContext', side_effect=get_integration_context)
     mocker.patch.object(SlackV3, 'send_slack_request_sync')
 
     # Arrange
     SlackV3.send_file_to_destinations(['channel'], {'name': 'name', 'path': 'yo'}, None)
-
-    args = SlackV3.send_slack_request_sync.call_args[1]
+    request_args = SlackV3.send_slack_request_sync.call_args[1]
 
     # Assert
     assert SlackV3.send_slack_request_sync.call_count == 1
-    assert 'http_verb' not in args
-    assert args['file_'] == 'yo'
-    assert args['body']['filename'] == 'name'
+    assert 'http_verb' not in request_args
+    assert request_args['file_upload_params']['file'] == 'yo'
+    assert request_args['file_upload_params']['filename'] == 'name'
+
+
+def test_send_file_to_destinations_sdk_client_args(mocker):
+    """
+    Given:
+        - Sending file to a specifc thread in a channel
```

Comment:
```suggestion
        - A file to send
```

Reply:
Improved doc string in commit 7c5b1ee65136fc655e92a3456301506de163b9e6